torch/testing/_internal/common_utils.py

```diff
@@ -919,6 +919,14 @@ def _check_module_exists(name: str) -> bool:
 # as we had before.  By default, we don't run these tests.
 TEST_WITH_CROSSREF = os.getenv('PYTORCH_TEST_WITH_CROSSREF', '0') == '1'
 
+try:
+    import torchvision.models  # noqa: F401
+    from torchvision.models import resnet18  # noqa: F401
+
+    HAS_TORCHVISION = True
+except ImportError:
+    HAS_TORCHVISION = False
+skipIfNoTorchVision = unittest.skipIf(not HAS_TORCHVISION, "no torchvision")
```

Comment:
What's going on here?

Reply:
These 5 lines are copy-pasted like 30 times in pytorch tests. I didnt yet refactor the uses but this is step one in doing that.